test/lib/rules/no-undef-class.js

```diff
@@ -358,6 +358,19 @@ ruleTester.run('no-undef-class', rule, {
       `,
       options: [{ camelCase: 'dashes-only' }],
     }),
+
+    /*
+      support id selectors
+    */
+    test({
+      code: `
+        import s from './id.scss';
+
+        export default Foo = () => (
+          <div id={s.scopedId}></div>
+        );
+      `,
+    }),
```

Comment:
Does this work with `no-unused-class` as well?

Reply:
Added tests for no-unused-class as well as an error test for no-undef-class